updater/internal/updater/templates.go

```diff
@@ -10,7 +10,7 @@ StartLimitBurst=5
 [Service]
 Type=simple
 User=root
-Group=observiq-otel-collector
+Group={{.Group}}
```

Comment:
Offline agent updates does not deploy the new "bdot" user or handle any migration logic. Instead, we can read the current user from the service and keep running with it.